config/manager/kustomization.yaml

```diff
@@ -1,5 +1,7 @@
 resources:
 - manager.yaml
+- config.yaml
+- ../samples/basicsecret.yaml
```

Comment:
no, don't include the Secret here at all. This kustomization.yaml represents what Numaplane will install by default.